resources/fixtures/config/titles.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  - Mr
+  - Mrs
+  - Ms
+  - Mx
+  - Doctor
```

Comment:
I think we should change this to 'Dr'. We can use a migration to update the user tables to replace 'Doctor' with 'Dr'. No point copying over something which was wrong to begin with